Light up bot/lightupbot.py

```diff
@@ -0,0 +1,87 @@
+import asyncio
+
+from viam.robot.client import RobotClient
+from viam.rpc.dial import Credentials, DialOptions
+from viam.components.camera import Camera
+from viam.components.types import CameraMimeType
+from viam.services.vision import VisionServiceClient, VisModelConfig, VisModelType, Detection
+from PIL import ImageDraw
+from kasa import Discover, SmartPlug
+
+ROBOT_URL = 'PUT YOUR URL HERE!!!'
```

Comment:
Your README says set these via environment variable, but if that is the expectation (and that is a good thing), these lines should look similar to:
https://github.com/viam-labs/devrel-demos/blob/b10c87dd89c4e5cba68cc6e8374076e3b4497d59/candy_machine/candy_machine.py#L11